File: Week 1/Javascript Qns/SingleThreaded/count.js
const startTime = Date.now();

let count = 0;
let sum = 0;

for (let i = 1; i <= 10000000000; i++) {
    count++;
    sum += i;
}

const endTime = Date.now();
const duration = (endTime - startTime) / 1000;

console.log(`Count: ${count}`);
console.log(`Sum: ${sum}`);
console.log(`Time taken: ${duration} seconds`);

File: Week 1/Javascript/SingleThreaded/count.go
package main

import (
    "fmt"
    "time"
)

func main() {
    startTime := time.Now()

    count := 0
    sum := 0

    for i := 1; i <= 10000000000; i++ {
        count++
        sum += i
    }

    endTime := time.Now()
    duration := endTime.Sub(startTime).Seconds()

    fmt.Printf("Count: %d\n", count)
    fmt.Printf("Sum: %d\n", sum)
    fmt.Printf("Time taken: %.2f seconds\n", duration)
}
